### PR TITLE
docs(releases): clarify allowed changes post rc

### PR DESCRIPTION
**Pull request by KellyMerrick**

### CODE REVIEW

File: content/community/contributing_guidelines/_index.md [hunk 1]
```diff
@@ -144,6 +144,10 @@ Currently, main releases only apply to [core repos](/docs/community/contributing
 
 - **Patches:** Patches will be released on an as needed basis for bug and security fixes. We increment the minor version.
 - **RC’s (pre-releases):** Prior to releasing a main version, we will have at least a week of rc releases for testing purposes.
+  - Once an RC is cut:
+    - No new features will be included.
```
Comment (kneal) [inline content/community/contributing_guidelines/_index.md]: No, new features makes me think large net new-ish work. 

What is the thought or feelings towards things like [refactors and improvements](https://go-vela.github.io/docs/community/contributing_guidelines/#development-workflow)?

Not sure there is a definition ether on [features vs enhancements](https://go-vela.github.io/docs/community/contributing_guidelines/#development-workflow). So, just curious about the thoughts.

Comment (KellyMerrick) [reply content/community/contributing_guidelines/_index.md]: That's good feedback. I'll add some definition regarding bug/enhancement/feature and update this section accordingly.

Comment (kneal) [reply content/community/contributing_guidelines/_index.md]: Thanks! 😄

Comment (KellyMerrick) [reply content/community/contributing_guidelines/_index.md]: Updates added.

**Review by kneal — APPROVED** [review]
LGTM 🐬

**Review by wass3r — APPROVED** [review]

**Pull request merged**